configs/osm_config.yaml

```diff
@@ -0,0 +1,401 @@
+osm_clean_columns:
+  substation:
+    bus_id: object
+    station_id: float
+    voltage: float
+    dc: bool
+    symbol: object
+    under_construction: bool
+    tag_substation: str
+    tag_area: str
+    lon: float
+    lat: float
+    country: str
+    geometry: object
+  generator:
+    id: object
+    tags.power: object
+    Type: object
+    Country: str
+    Area: object
+    name: object
+    tags.generator:type: object
+    tags.generator:method: object
+    tags.generator:source: object
+    power_output_MW: object
+    geometry: object
+  line:
+    line_id: object
+    bus0: object
+    bus1: object
+    voltage: float
+    circuits: float
+    length: float
+    underground: bool
+    under_construction: bool
+    tag_type: str
+    tag_frequency: float
+    dc: bool
+    country: object
+    geometry: object
+
+continents:
+  LA: NorthAmerica
+  SA: SouthAmerica
+  AS: Asia
+  OC: Oceania
+  AF: Africa
+  EU: Europe
+
+world_iso:
```

Comment:
Awesome!
In the latest PR, there are few countries that have been uncommented.
Could you please include them?

Link: https://github.com/pypsa-meets-earth/pypsa-earth/pull/817/files
See changes to config_osm_data

Reply:
done.